src/test/java/com/thealgorithms/datastructures/graphs/WelshPowellAlgorithmTest.java

```diff
@@ -0,0 +1,63 @@
+package com.thealgorithms.datastructures.graphs;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.thealgorithms.datastructures.graphs.WelshPowell.Graph;
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+
+class WelshPowellAlgorithmTest {
+
+    @Test
+    void testSimpleGraph() {
+        int[][] edges = {{0, 1}, {1, 2}, {2, 3}};
+        Graph graph = WelshPowell.makeGraph(4, edges);
```

Comment:
```suggestion
        final var graph = WelshPowell.makeGraph(4, new int[][] {{0, 1}, {1, 2}, {2, 3}});
```
There are two things:
- I do not see a point of defining `edges` explicitly,
- note the `var`: thanks to it, we can skip the `import com.thealgorithms.datastructures.graphs.WelshPowell.Graph;`. `final` is just a nice addition.